### ISSUE TITLE
Prod build error 

### ISSUE THREAD

**Issue by learnuser521** [recogizer/angular-gauge-chart#27]
Building the library with ng serve --prod fails 

ERROR in main.......js from Terser
Unexpected token: punc ()) [main.......js:]

**Comment by mmaltsev** [comment]
Hi @learnuser521,

Unfortunately, our project doesn't support `ng serve --prod`. You can see the scripts we are working with in our `package.json` file.

**Comment by learnuser521** [comment]
The error is still occurs when the library is included in an app and the app is built in prod mode

**Comment by maihannijat** [comment]
@mmaltsev Do you mean the project is not compatible with Angular? I doubt if someone build an Angular app without `--prod` flag for production.

**Comment by cherurg** [comment]
@maihannijat it is a library, not an angular app.  We use usual `ng serve` command to run an app that hosts a dev environment for the library.  
The library can be built using usual `ng build angular-gauge-chart` command.

**Comment by maihannijat** [comment]
@cherurg Thanks for the reply.

I used the package same way the documentation says on the first page. I am able to build it in Angular 7 with production flag but my project does not build with production build if I have:
```
@NgModule({
  declarations: [XYZ],
  imports: [
    ...
    GaugeChartModule,
  ]
})
```
Is there any other way to use it? Am I doing something wrong?

**Comment by cherurg** [comment]
@learnuser521 as @maihannijat pointed out in #28, the issue is present in many cases. It's fixed in `angular-gauge-chart@0.7.2`.
